[package] mysql-connector-cpp/9.2.0: Unable to build on Visual Studio 2022

### Description

Hi, there is an error for building mysql-connector-cpp for version 9.2.0

The error is **LINK : fatal error LNK1104: cannot open file 'mysqlcppconnx-static.lib'**

Appreciate if the team can update the recipe. Thanks!

### Package and Environment Details

* Package Name/Version: **mysql-connector-cpp/9.2.0**
* Operating System+version: **Windows 11**
* Compiler+version: **MSVC**
* Conan version: **conan 2.20.1**
* Python version: **Python 3.13.9**


### Conan profile

[settings]
arch=x86_64
build_type=Release
compiler=msvc
compiler.cppstd=14
compiler.runtime=dynamic
compiler.version=194
os=Windows

### Steps to reproduce

conan install using Visual Studio 

### Logs

Rebuild started at 9:11 am...
1>------ Rebuild All started: Project: ConsoleApplication1, Configuration: Debug x64 ------
1>Checking changes in conandata.yml
1>Checking changes in conanfile.py
1>No changes detected, skipping conan install...
1>ConsoleApplication1.cpp
1>LINK : fatal error LNK1104: cannot open file 'mysqlcppconnx-static.lib'
1>Done building project "ConsoleApplication1.vcxproj" -- FAILED.
========== Rebuild All: 0 succeeded, 1 failed, 0 skipped ==========
========== Rebuild completed at 9:11 am and took 01.641 seconds ==========

--- a/recipes/mysql-connector-cpp/all/conanfile.py
+++ b/recipes/mysql-connector-cpp/all/conanfile.py
@@ -118,10 +118,10 @@ class MysqlConnectorCppConan(ConanFile):
             if is_msvc_static_runtime(self):
                 lib += "-mt"
         self.cpp_info.libs = [lib]
+        self.cpp_info.libdirs = ["lib"] if self.settings.build_type == "Release" else [os.path.join("lib", "debug")]
 
         if self.settings.os == "Windows":
-            libdirs = [os.path.join("lib", "vs14") if self.settings.build_type == "Release" else os.path.join("lib", "debug", "vs14")]
-            self.cpp_info.libdirs = libdirs
+            self.cpp_info.libdirs = [os.path.join("lib", "vs14")] if self.settings.build_type == "Release" else [os.path.join("lib", "debug", "vs14")]
             self.cpp_info.bindirs = ["lib"]
             self.cpp_info.system_libs.extend(["dnsapi", "ws2_32"])
         if self.settings.os in ["Linux", "FreeBSD"]:

